dashboards-observability/public/components/event_analytics/explorer/log_patterns/pattern_detail_flyout.tsx

```diff
@@ -0,0 +1,219 @@
+    },
+    {
+      title: 'Pattern length:',
+      description: pattern.puncSignature.length,
+    },
+    {
+      title: 'Earliest time:',
+      description: moment(new Date(pattern.firstTimestamp)).format(UI_DATE_FORMAT),
+    },
+    {
+      title: 'Recent time:',
+      description: moment(new Date(pattern.lastTimestamp)).format(UI_DATE_FORMAT),
+    },
+  ];
+
+  const patternDetailsList = (
+    <EuiFlexGroup gutterSize="none">
+      {patternDetails.map((detail) => {
+        return (
+          <EuiFlexGroup gutterSize="s" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiText size="xs">
+                <h3>{detail.title}</h3>
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText size="xs">
+                <p>{detail.description}</p>
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        );
+      })}
+    </EuiFlexGroup>
+  );
+
+  const search = {
+    box: {
+      incremental: true,
+    },
+  };
+
+  const dummyEventsData = [
+    {
+      timestamp: Date.now(),
+      message:
+        '[nginx][access] 50.116.6.118 0.006 "GET /node_modules/@kbn/ui-framework/dist/kui_light.css HTTP/1.1" 200 13313',
+    },
+    {
+      timestamp: Date.now(),
+      message:
+        '[mongodb.log][I] received client metadata from 127.0.0.1:39128 conn10128200: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "3.6.22" }, os: { type: "Linux", name: "PRETTY_NAME="Debian GNU/Linux 9 (stretch)"", ',
+    },
+    {
+      timestamp: Date.now(),
+      message:
+        '[nginx][access] 50.116.6.118 0.006 "GET /node_modules/@kbn/ui-framework/dist/kui_light.css HTTP/1.1" 200 13313',
+    },
+    {
+      timestamp: Date.now(),
+      message:
+        '[mongodb.log][I] received client metadata from 127.0.0.1:39128 conn10128200: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "3.6.22" }, os: { type: "Linux", name: "PRETTY_NAME="Debian GNU/Linux 9 (stretch)"", ',
+    },
+    {
+      timestamp: Date.now(),
+      message:
+        '[nginx][access] 50.116.6.118 0.006 "GET /node_modules/@kbn/ui-framework/dist/kui_light.css HTTP/1.1" 200 13313',
+    },
+    {
+      timestamp: Date.now(),
+      message:
+        '[mongodb.log][I] received client metadata from 127.0.0.1:39128 conn10128200: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "3.6.22" }, os: { type: "Linux", name: "PRETTY_NAME="Debian GNU/Linux 9 (stretch)"", ',
+    },
+  ];
+
+  const sorting = {
+    sort: {
+      field: 'timestamp',
+      direction: 'desc' as Direction,
+    },
+    allowNeutralSort: true,
+    enableAllColumns: true,
+  };
+
+  const eventsTableColumns = [
+    {
+      field: 'timestamp',
+      name: 'Timestamp',
+      sortable: true,
+      width: '15%',
+      render: (value: any) => (
```

Comment:
np: it looks like `value` here is an input to be cast as a `Date` type object, can we be more specific with its type than `any`?